resources/facts.txt

```diff
@@ -68,3 +68,33 @@ Several ships have wrecked on Disappointment Island.
 The Eiffel Tower is repainted every seven years.
 In 1992, a shipping crate containing 29,000 rubber ducks, beavers, turtles, and frogs fell overboard. They washed up around the world for the next 20 years, and have been used to track ocean currents.
 A group of lemurs is called a conspiracy.
+The first person convicted of speeding was going eight mph.
+"New car smell" is the scent of dozens of chemicals.
+Some sea snakes can breathe through their skin.
+The heads on Easter Island have bodies.
+The moon has moonquakes.
+Goosebumps are meant to ward off predators.
+Humans are the only animals that blush.
+The wood frog can hold its pee for up to eight months.
```

Comment:
Add to the end "It does so by recycling urea to stay warm during winter hibernation."